docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
-name: dj
```

Comment:
Did you find this line causing issues on your side as well?

Reply:
I didn't but I figured there's no reason to have it if it causes issues in some environments. From what I can tell it's just a display value.